site/en/install/pip.md

```diff
@@ -193,7 +193,26 @@ The following NVIDIA® software are only required for GPU support.
     nvidia-smi
     ```
 
-    ### 3. Install TensorFlow
+    ### 3. Create a virtual environment with [venv](https://docs.python.org/3/library/venv.html){:.external}
+      
+    The venv module is part of Python’s standard library and is the officially recommended way to create virtual environments. 
+    
+    Navigate to your desired virtual environments directory and create a new venv environment named `tf` with the following command.
+
+    ```bash
+    python3 -m venv tf 
+    ```
+
+    You can activate and deactivate it with the following commands.
+
+    ```bash
+    source tf/bin/activate
+    deactivate
```

Comment:
Can you remove `deactivate`?

Reply:
I want to make sure that I understand the situation correctly. Which of the following two situation is correct?
- If the issue doesn't happen for 2.17.0RC0, yes please remove the instructions.
- If the issue happens for both 2.17.0RC0 and 2.16, we can wait for the GPU team to take a look at https://github.com/tensorflow/tensorflow/issues/63362 and see if the can send a fix for both 2.16.2 and 2.17.0 release.

Reply:
The only difference is that on version 2.17.0.rc0 you only need the symlinks to nvidia libs in order to utilize GPUs while on version 2.16.1 you should in addition to creating symlinks to NVIDIA libs create symlink to ptxas. Eitherway the command `pip install tensorflow[and-cuda]` is not enough to utilize GPUs.

Reply:
@learning-to-play unfortunately the latest release namely TensorFlow `2.16.2` does not fix the `ptxas` bug. When running a training script the user gets the error:

> ptxas returned an error during compilation of ptx to sass: 'INTERNAL: ptxas 12.3.103 has a bug that we think can affect XLA. Please use a different version.'  If the error message indicates that a file could not be written, please verify that sufficient filesystem space is provided.
> Aborted (core dumped)